src/javascript/app/Modules/Trading/Containers/purchase.jsx

```diff
@@ -22,16 +22,15 @@ const Purchase = ({
     validation_errors,
 }) => {
     const is_high_low = /high_low/.test(contract_type.toLowerCase());
-    const isLoading = info => {
-        const has_validation_error = Object.values(validation_errors).some(e => e.length);
-        return !has_validation_error && !info.has_error && !info.id;
-    };
+    const has_validation_error = Object.values(validation_errors).some(e => e.length);
+
+    const isLoading = info => !has_validation_error && !info.has_error && !info.id;
 
     const components = [];
     Object.keys(trade_types).map((type, index) => {
         const info              = proposal_info[type] || {};
         const is_disabled       = !is_purchase_enabled || !is_trade_enabled || !info.id || !is_client_allowed_to_visit;
-        const is_proposal_error = info.has_error && !info.has_error_details;
+        const is_proposal_error = has_validation_error || (info.has_error && !info.has_error_details);
```

Comment:
@cakasuma for errors that we show on validation field, we don't want to show on purchase button, this is wrong